src/runtime/gateways/file/node-filesystem.ts

```diff
@@ -17,7 +16,7 @@ export class NodeFileSystem implements SysFileSystem {
   };
 
   async start(): Promise<SysFileSystem> {
-    this.fs = await import("node:fs/promises");
+    this.fs = await import("fs/promises");
```

Comment:
👍 💯

Reply:
lets use node: to ensure that deno will fail -;)